hooks/post_gen_project.sh

```diff
@@ -1,47 +0,0 @@
-#!/usr/bin/env bash
-
-if ! make venv; then
```

Comment:
Wow, Vlad!  Many thanks for all your work here!!  I am wondering though why you removed the secondary initialization stuff (creating the virtual environment, installing the requirements, and calling build).  Does it cause a problem?

Reply:
Not a problem, but should be left up to user. Maybe they don't want a venv or want it elsewhere. It should really be left up to the user.